lib/init.php

```diff
@@ -2,17 +2,22 @@
 /**
  * Prepare and initialize the Beans framework.
  *
- * @package Initialize
+ * @package Beans\Framework
+ *
+ * @since   1.0.0
  */
 
 add_action( 'beans_init', 'beans_define_constants', -1 );
 /**
  * Define constants.
  *
+ * @since 1.0.0
  * @ignore
+ * @access private
```

Comment:
The access is not private.  Go ahead and remove the `@access` tag.  We do want to ignore it though for the automatic doc generator.